transformer_engine/jax/cpp_extensions/custom_call.py

```diff
@@ -17,8 +19,25 @@
     # version, so we still need this import.
     pass
 
+
+class CustomCallAPIVersion(IntEnum):
+    """Enum for selecting between old and new custom call registration API"""
+
+    OPAQUE = 0
+    FFI = 1
+
+
 for _name, _value in transformer_engine_jax.registrations().items():
-    xla_client.register_custom_call_target(_name, _value, platform="CUDA")
+    if _name.endswith("_ffi"):
+        if is_ffi_enabled():
+            # COMMAND_BUFFER_COMPATIBLE i.e. cudaGraph enabled by default
+            xla_client.register_custom_call_target(
+                _name, _value, platform="CUDA", api_version=CustomCallAPIVersion.FFI.value
```

Comment:
This don't seem to pass the cudaGraph traits.
But the comment and the PR description tell it enable cudaGraph.
Do this need a PR to enable it?
https://github.com/openxla/xla/blob/de99c1bca9ac1ca5faee3c09f46f56b2c0060f89/xla/python/xla_client.py#L589